src/definitions/modules/calendar.js

```diff
@@ -1176,6 +1176,7 @@ $.fn.calendar.settings = {
   centuryBreak       : 60,         // starting short year until 99 where it will be assumed to belong to the last century
   currentCentury     : 2000,       // century to be added to 2-digit years (00 to {centuryBreak}-1)
   selectAdjacentDays : false,     // The calendar can show dates from adjacent month. These adjacent month dates can also be made selectable.
+  monthParseLength   : 3,
```

Comment:
I thinkt adding new setting may complicate calendar modules for non-English users and for multi-lingual apps.
Instead of new settings, how about making the parsing logic works with any-length ?

Reply:
Agree 👍 I removed the extra setting now and also removed the length calculation limit now by fb81e3e62089f6f0184c77b0fce96963a2fa9018, much easier. Thx for the suggestion!